examples/benchmark.py

```diff
@@ -81,7 +82,13 @@ def source(t, f0):
 data.set_receiver_pos(receiver_coords)
 data.set_shape(nt, 30)
 # A Forward propagation example
-jit_obj = AcousticWave2D_cg(model1, data, create_dm, nbpml=nbpml, t_order=2, s_order=2)
+
+# path at which numpy memmap file are stored
+cache_disk_path = "/tmp/devito_disk"
+if not os.path.exists(cache_disk_path):
+    os.makedirs(cache_disk_path)
+
+jit_obj = AcousticWave2D_cg(model1, data, create_dm, nbpml=nbpml, t_order=2, s_order=2, disk_path=cache_disk_path)
```

Comment:
@mlange05 These are the kind of parameters you wouldn't have to pass if you had a config file.